Typo fixes 2025-04-09 (#497)
eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework: docs/annexes/annex-2/annex-2-high-level-requirements.md

The PID Provider or Attestation Provider then asks the Wallet Unit to create a key
pair for its new PID or attestation, and to prove that both this new private key and the
private key corresponding to public key in the WUA are in possession of the Wallet Unit.

> A topic related to the WUA is the following. It would be useful if the Wallet Unit were able provide a proof that the PID or attestation private key is protected by the same WSCA as the WUA private

Reviewer: ```suggestion
A topic related to the WUA is the following. It would be useful for the Wallet Unit to be able to provide a proof that the PID or attestation private key is protected by the same WSCA as the WUA private
```

| **Index** | **Requirement specification** |
|-----------|-----------------|

> | EDP_01 | A Wallet Unit SHALL enable an Attestation Provider to optionally express embedded disclosure policies for a QEAA, PuB-EAA, or non-qualified EAA. *Note: The [European Digital Identity Regulation] does not contain a requirement for PIDs to be able to contain an embedded disclosure policy.* |

Reviewer 2: ```suggestion
| EDP_01 | A Wallet Unit SHALL enable an Attestation Provider to optionally express an embedded disclosure policy for a QEAA, PuB-EAA, or non-qualified EAA. *Note: The [European Digital Identity Regulation] does not contain a requirement for PIDs to be able to contain an embedded disclosure policy.* |
```

Reviewer 2: An Attestation Provider can have only one EDP per attestation type. At least, I would not like to suggest that there can be multiple...
